app/components/Map.js

```diff
@@ -107,7 +115,7 @@ const styles = StyleSheet.create({
     marginBottom: 5,
   },
   map: {
-    height: Dimensions.get('window').height/2,
+    height: Dimensions.get('window').height/1.5,
```

Comment:
Is it possible to use the flex container to make it so that the map fills up everything it can except for the two buttons?
Do we want the button to be outside of the map or kind of floating on top of it?